controls/radchartview/populating-with-data/chartdatasource.md

```diff
@@ -239,22 +219,36 @@ Below you'll find a sample where this is demonstrated (the datasource is the sam
 	        return new ChartMinFunction();
 	    }
 	}
-	{{endregion}}
-
-
-
-#### __VB.NET__
+{{endregion}}
 
+#### __[VB.NET] Example 5: Create custom series and overriding its aggregate function__
 {{region radchart-chartdatasource_6}}
 	Public Class MyBarSeries
 		Inherits BarSeries
 		Protected Overrides Function GetValueAggregateFunction() As ChartAggregateFunction
 			Return New ChartMinFunction()
 		End Function
 	End Class
-	{{endregion}}
-
-
+{{endregion}}
 
 The result is below:
 ![Rad Chart View-chart chartdatasource 01](images/RadChartView-chart_chartdatasource_01.PNG)
+
+## DataPointSamplingInfo
+
+The chart's [DataPoint]({%slug radchartview-getting-started-data-point%}) models which are used to plot the data items in the control, expose a __DataItem__ property which holds a reference to view model behind the point. However, when sampling (ChartDataSource) is used, several data point view models (data items) are combined into a single DataPoint. In this case the __DataItem__ property contains an object of type __DataPointSamplingInfo__. This object has information about the data items that are combined and the results returned by the aggregate function. 
+
+__DataPointSamplingInfo__ is IEnumerable and you can get the aggregated data items using its indexer

```

Comment:
__DataPointSamplingInfo__ is IEnumerable and you can get the aggregated data items using its indexer(.)